src/auth/auth-repository.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@nestjs/common';
+import { InjectModel } from '@nestjs/mongoose';
+import { Model } from 'mongoose';
+import { User } from 'src/user.entity';
+
+@Injectable()
+export class AuthRepository {
+  constructor(@InjectModel('Users') private UserModel: Model<any>) {}
+  async findUser(email: string): Promise<any> {
+    return this.UserModel.find({ email: email });
```

Comment:
Is this a helper method?? If yes then findUser should also be based on userId. Let's discuss this point.